test/recipes/25-test_req.t

```diff
@@ -32,17 +42,10 @@ ok(!run(app(["openssl", "req", "-new", "-addext", $val, "-addext", $val])));
 ok(!run(app(["openssl", "req", "-new", "-addext", $val, "-addext", $val2])));
 ok(!run(app(["openssl", "req", "-new", "-addext", $val, "-addext", $val3])));
 ok(!run(app(["openssl", "req", "-new", "-addext", $val2, "-addext", $val3])));
+ok(run(app(["openssl", "req", "-config", srctop_file("test", "test.cnf"),
+		"-addext", $val, @req_new, "-out", "testreq.pem"])));
```

Comment:
No, all the working arguments should be passed even to negative tests. Because if it fails to detect duplicated -addext, test should fail, i.e. request succeed.